lib/placeNewRelic.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs')
```

Comment:
minor: but shouldn't that use all "import" not require? afaik the proper, new way of doing things is import - or is there a technical reason to use require instead?